client/src/containers/interventions/new/step1/component.tsx

```diff
@@ -8,104 +8,65 @@ import { analysis, setFilter } from 'store/features/analysis';
 // components
 import Select from 'components/select';
 
+import Materials from 'containers/analysis-visualization/analysis-filters/materials/component';
+import Suppliers from 'containers/analysis-visualization/analysis-filters/suppliers/component';
+import OriginRegions from 'containers/analysis-visualization/analysis-filters/origin-regions/component';
+
 // types
 import { SelectOptions, SelectOption } from 'components/select/types';
+import type { AnalysisState } from 'store/features/analysis';
+
+const businesses = ['business1', 'business2', 'business3'];
+const yearCompletions = [2001, 2015, 2020];
+const interventionTypes = [
+  'Source from a new supplier or location',
+  'Change production efficiency',
+  'Switch to a new material',
+];
 
 const Step1 = () => {
   const dispatch = useAppDispatch();
   //const [isOpen, setIsOpen] = useState<boolean>(false);
   const { filters } = useAppSelector(analysis);
-  const { materials: materialsData } = filters;
-
-  const onChange = useCallback((key: string, value: string | number) => {
-    console.log('onChnage', key, value);
-  }, []);
-  const materials = ['material1', 'material2', 'material3'];
-  const businesses = ['business1', 'business2', 'business3'];
-  const supliers = ['supplier1', 'supplier2', 'supplier3'];
-  const sourcingRegions = ['sourcingRegion1', 'sourcingRegion2', 'sourcingRegion3'];
-  const yearCompletions = [2001, 2015, 2020];
-  const interventionTypes = [
-    'Source from a new supplier or location',
-    'Change production efficiency',
-    'Switch to a new material',
-  ];
 
   // const { data: materials, isLoading: isLoadingMaterials } = useMaterials();
```

Comment:
why did you comment these lines? The data should come from those hooks, also the hardcoded data